### PR TITLE
Carousel file- swiper

### PR DESCRIPTION
**Pull request by shirrotem**

### CODE REVIEW

File: src/Card.scss [hunk 1]
```diff
@@ -57,10 +66,87 @@
...
     }
+
+    //Desktop
+    @media (min-width: 901px) {
+      display: flex;
+      padding: 24px 0px 8px 0px;
+      flex-direction: column;
+      align-items: center;
+      gap: 8px;
+      flex: 1 0 0;
```
Comment (sivanbasa93) [inline src/Card.scss]: flex: 1 0 0;
 align-self: stretch;
 
 check if relevant

File: src/Card.scss [hunk 2]
```diff
@@ -57,10 +66,87 @@
...
+          width: 14px;
+          height: 19px;   
+        }
+  
+        .card-title {
+          text-align: center;
+          font-size: 40px;
+          font-style: normal;
+          font-weight: 400;
+          line-height: 47px; /* 117.5% */
```
Comment (sivanbasa93) [inline src/Card.scss]: please delete comment

File: src/Card.scss [hunk 3]
```diff
@@ -57,10 +66,87 @@
...
+          letter-spacing: 2.67px;
+        }
+    
+        .card-subtitle {
+          text-align: center;
+          font-size: 24px;
+          font-style: normal;
+          font-weight: 200;
+          line-height: 30px; /* 125% */
```
Comment (sivanbasa93) [inline src/Card.scss]: check if relevant :
          font-style: normal;
and please remove comment

File: src/Card.scss [hunk 4]
```diff
@@ -57,10 +66,87 @@
...
+          display: flex;
+          justify-content: center;
+          align-items: center;
+          gap: 8px;
+  
+  
+          .card-price{
+              color: #000;
+              text-align: center;
+              font-family: Helvetica Neue;
```
Comment (sivanbasa93) [inline src/Card.scss]: why not using with variable for font family?

File: src/Card.scss [hunk 5]
```diff
@@ -57,10 +66,87 @@
...
+              font-size: 24px;
+              font-style: normal;
```
Comment (sivanbasa93) [inline src/Card.scss]: check if relevant 
              font-style: normal;

File: src/Card.scss [hunk 6]
```diff
@@ -57,10 +66,87 @@
...
+        .card-title {
+          text-align: center;
+          font-size: 40px;
+          font-style: normal;
+          font-weight: 400;
+          line-height: 47px; /* 117.5% */
+          letter-spacing: 2.67px;
+        }
+    
+        .card-subtitle {
```
Comment (sivanbasa93) [inline src/Card.scss]: i see that .card-subtitle and .card-price contain same fields. 
maybe think of something gives reuse code and not duplicate code?

File: src/Contact.scss [hunk 7]
```diff
@@ -3,7 +3,7 @@
     padding-inline-start: 0px;
     background-color: #FFF;
     flex-shrink: 0;
-    width: 377px;
+    // width: 377px;
```
Comment (sivanbasa93) [inline src/Contact.scss]: please remove comment

File: src/Contact.scss [hunk 8]
```diff
@@ -17,11 +17,12 @@
     @media (min-width: 901px) {
         display: flex;
         flex-direction: row;
```
Comment (sivanbasa93) [inline src/Contact.scss:18]: if we say display:flex 
by default 
        flex-direction: row; 

so, you don't have to write it

File: src/Contact.scss [hunk 9]
```diff
@@ -17,11 +17,12 @@
     @media (min-width: 901px) {
         display: flex;
         flex-direction: row; 
-        justify-content: space-around;
+        justify-content: center;
         align-items: center;
         gap: 10px;
         align-self: stretch;
```
Comment (sivanbasa93) [inline src/Contact.scss:22]: check if 
        align-self: stretch;
relevant

File: src/pages/home-page/Carousel.scss [hunk 10]
```diff
@@ -0,0 +1,20 @@
+.allRestaurants {
+    display: flex;
+    align-items: center;
+    justify-content: flex-start; 
+    text-align: right;
+    font-family: Helvetica Neue;
```
Comment (sivanbasa93) [inline src/pages/home-page/Carousel.scss]: variable

File: src/pages/home-page/Carousel.tsx [hunk 11]
```diff
@@ -0,0 +1,57 @@
...
+import 'swiper/css';
+import 'swiper/css/pagination';
+import Card from "../../Card";
+import {FC} from 'react';
+import useMobileCheck from "../../shared/hooks/useMobileCheck";
+import arrowsIcon from "../../assets/icons/arrows.svg";
+import './Carousel.scss'
+
+
+interface CarouselProps {
```
Comment (sivanbasa93) [inline src/pages/home-page/Carousel.tsx]: please move to directory : types/interfaces
and put this interface in separate file

File: src/pages/home-page/HomePage.scss [hunk 12]
```diff
@@ -0,0 +1,18 @@
+.titleSwiper{
+    font-size: 18px;
+    font-style: normal;
+    font-weight: 200;
+    line-height: 24px;
+    letter-spacing: 1.25px;
+    text-transform: uppercase;
```
Comment (sivanbasa93) [inline src/pages/home-page/HomePage.scss]: check if relevant

File: src/pages/home-page/HomePage.scss [hunk 13]
```diff
@@ -0,0 +1,18 @@
...
+}
+
+.titleSwiperChef{
+    font-size: 16px;
+    font-style: normal;
+    font-weight: 200;
+    line-height: 25px;
+    letter-spacing: 1.25px;
+    text-transform: uppercase;
```
Comment (sivanbasa93) [inline src/pages/home-page/HomePage.scss]: check if relevant

File: src/data/MockData.tsx [hunk 14]
```diff
@@ -1,6 +1,14 @@
...
+import padKiMaoImg from "../assets/images/padKiMaoImg.png";
+import garbanzoFritoImg from "../assets/images/garbanzoFritoImg.png";
+import smokedPizzaImg from "../assets/images/smokedPizzaImg.png";
+import spicyIcon from "../assets/icons/spicy.svg";
+import veganIcon from "../assets/icons/vegan.svg";
+import onzaImg from "../assets/images/onzaImg.png";
+import kitchenMarketImg from "../assets/images/kitchenMarket.png";
+import mashyaImg from "../assets/images/mashyaImg.png";
 
 const data = {
```
Comment (sivanbasa93) [inline src/data/MockData.tsx]: please create type for data

File: src/Card.scss (hunk 1 above)
Comment (shirrotem) [reply src/Card.scss]: fixed

File: src/Card.scss (hunk 2 above)
Comment (shirrotem) [reply src/Card.scss]: fixed

File: src/Card.scss (hunk 3 above)
Comment (shirrotem) [reply src/Card.scss]: fixed

File: src/Card.scss (hunk 5 above)
Comment (shirrotem) [reply src/Card.scss]: fixed

File: src/pages/home-page/Carousel.scss (hunk 10 above)
Comment (shirrotem) [reply src/pages/home-page/Carousel.scss]: fixed

File: src/pages/home-page/HomePage.scss (hunk 12 above)
Comment (shirrotem) [reply src/pages/home-page/HomePage.scss]: fixed

File: src/pages/home-page/HomePage.scss (hunk 13 above)
Comment (shirrotem) [reply src/pages/home-page/HomePage.scss]: fixed

File: src/Card.scss (hunk 4 above)
Comment (shirrotem) [reply src/Card.scss]: fixed

File: src/data/MockData.tsx (hunk 14 above)
Comment (shirrotem) [reply src/data/MockData.tsx]: fixed

File: src/pages/home-page/Carousel.tsx (hunk 11 above)
Comment (shirrotem) [reply src/pages/home-page/Carousel.tsx]: fixed

File: src/shared/components/Contact/Contact.scss [hunk 15]
```diff
@@ -3,7 +3,7 @@
     padding-inline-start: 0px;
     background-color: #FFF;
     flex-shrink: 0;
-    width: 377px;
+    // width: 377px;
```
Comment (MoranOshiaMoveo) [inline src/shared/components/Contact/Contact.scss]: Remove comments

**Review by MoranOshiaMoveo — CHANGES_REQUESTED** [review]

File: src/shared/components/Contact/Contact.scss (hunk 15 above)
Comment (shirrotem) [reply src/shared/components/Contact/Contact.scss]: fixed

**Pull request merged**